specs/receive_funds/01_main.rst

```diff
@@ -0,0 +1,89 @@
++-------+---------------+-----------+------------+
+| issue | title         | author    | created    |
++-------+---------------+-----------+------------+
+| 68_   | Receive funds | tschubotz | 2019-01-17 |
++-------+---------------+-----------+------------+
+
+.. _68: https://github.com/gnosis/safe/issues/68
+
+Receive funds
+===============
+
+1. `Main`_
+2. `User Interface`_
+3. `External Communication`_
+4. Other_
+
+.. _Main:
+
+.. contents:: Table of Contents
+    :depth: 3
+
+1. Problem Definition
+---------------------
+
+- User wants to receive funds
+- User should be able to share their wallet address via different ways
+    - Via QR code for scanning.
+    - Copy to clipboard.
+    - Via other services on their phone.
+- There should be a way to double check if the other person has the
+  correct shared address.
+
+2. Inputs
+-----------
+
+- Address of the currently selected Safe.
+
+3. Outputs
+------------
+
+- Address is clearly visible.
```

Comment:
I find clearly is a very unspecific description. 
-> I assume we want to display the address completely (all characters) and with Checksum and with our (somewhere) defined address highlighting